int32 id
string name
geometry_msgs/Point[4] corner